proto/storage/usage.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "storage";
+
+message Usage {
+    google.protobuf.Timestamp timestamp = 1 [(gogoproto.moretags) = 'sql:"pk" search:"Timestamp"'];
+    int64 num_nodes = 2;
```

Comment:
I would add search from the start to both num node and num cpu units. Then if you want to get the max of all time you could just run two queries and sort by desc on those two units

Reply:
We mostly need to query by time range (last year, last quarter, random). All time is only a specific case of the general requirement. Not sure it is worth to complicate the code for this.